### PR TITLE
Approval Workflows Docs

### PR DESCRIPTION
**Pull request by katemacfarlane**
Approval Workflows docs without some images for now

### CODE REVIEW

**Review by kaushalkapasi — APPROVED** [review]

File: docs/extras/approval-workflows/approval-workflows.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+3. You may also choose whether Publishers can skip the approval process and save changes directly without requiring review from another teammate. 
+4. Click `Save` and Approval Workflows will now be active for that project. 
+
+![Project Settings for Approval Workflows](/july-2024-project-settings-approval.png)
+
+## Requesting a Review for a Change Request
+
+Once enabled, all team members will require approval on Feature changes in <b> all environments </b> from a Publisher or Owner within your DevCycle organization. 
+
+1. Onnce you've made your desired Feature changes, click on the `Submit Change Request` button at the top-right corner of the Feature page.
```
Comment (suthar26) [inline docs/extras/approval-workflows/approval-workflows.md:41]:
```suggestion
1. Once you've made your desired Feature changes, click on the `Submit Change Request` button at the top-right corner of the Feature page. 
```

File: docs/extras/approval-workflows/approval-workflows.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+1. Navigate to your chosen Project's settings page and find the Approval Workflows section. 
+2. Select `Enabled` from the dropdown. 
+3. You may also choose whether Publishers can skip the approval process and save changes directly without requiring review from another teammate. 
+4. Click `Save` and Approval Workflows will now be active for that project. 
+
+![Project Settings for Approval Workflows](/july-2024-project-settings-approval.png)
+
+## Requesting a Review for a Change Request
+
+Once enabled, all team members will require approval on Feature changes in <b> all environments </b> from a Publisher or Owner within your DevCycle organization.
```
Comment (suthar26) [inline docs/extras/approval-workflows/approval-workflows.md:39]: this may need a reword?

File: docs/extras/approval-workflows/approval-workflows.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+sidebar_position: 1
+---
+
+# Approval Workflows
+
+:::info
+Approval Workflows are an Enterprise feature. To learn more, read about our [pricing](https://devcycle.com/pricing). To upgrade your plan, please contact [Sales](mailto:sales@devcycle.com) or your Account Manager. 
+:::
+
+With Approval Workflows, when a DevCycle user wants to change a Feature, they must request review and approval from a Publisher within their DevCycle organization. Approval Workflows give people greater visibility on Feature changes and act as a safeguard for your team. These review-style approvals mirror common code review workflows, such as pull request (PR) reviews in GitHub.
```
Comment (jsalaber) [inline docs/extras/approval-workflows/approval-workflows.md:12]: `they must request a review and approval`

File: docs/extras/approval-workflows/approval-workflows.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+![Project Settings for Approval Workflows](/july-2024-project-settings-approval.png)
+
+## Requesting a Review for a Change Request
+
+Once enabled, all team members will require approval on Feature changes in <b> all environments </b> from a Publisher or Owner within your DevCycle organization. 
+
+1. Onnce you've made your desired Feature changes, click on the `Submit Change Request` button at the top-right corner of the Feature page. 
+2. Review and confirm the changes displayed in the Review Change modal.
+3. Enter a Description to add details that help your reviewers understand the changes you made.
```
Comment (suthar26) [inline docs/extras/approval-workflows/approval-workflows.md:43]: maybe reword? 
```suggestion
3. Enter a description that help your reviewers understand the changes you made.
```

File: docs/extras/approval-workflows/approval-workflows.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+4. Select one or more reviewers from the Reviewers menu and click Submit.
```
Comment (suthar26) [inline docs/extras/approval-workflows/approval-workflows.md:44]:
```suggestion
4. Select one or more reviewers from the Reviewers list and click Submit. 
```

File: docs/extras/approval-workflows/approval-workflows.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+After the Change Request is submitted, the Feature form will be locked for <b> all users </b> until the active Change Request is approved, rejected, or canceled. 
+
+Members who do not have permission to approve requests can still view the Change Request at the top of the Feature form. 
+
+![Active Change Request on Feature Form](/july-2024-active-change-request-feature-form.png)
+
+## Force Applying a Change Request as a Publisher
+
+If the <b> "Publishers can skip the review process" </b> setting has been turned on for your Project, Publishers, and Owners will be able to Force Apply their Change Request and bypass the review process. This feature is intended to reduce friction for users who have publishing permissions.
```
Comment (suthar26) [inline docs/extras/approval-workflows/approval-workflows.md:56]:
```suggestion
Force applying changes to bypass the review process for Publishers is only available if the <b> "Publishers can skip the review process" </b> setting has been turned on for your Project. This feature is intended to reduce friction for users who have publishing permissions. 
```

File: docs/extras/approval-workflows/approval-workflows.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+
+![Active Change Request on Feature Form](/july-2024-active-change-request-feature-form.png)
+
+## Force Applying a Change Request as a Publisher
+
+If the <b> "Publishers can skip the review process" </b> setting has been turned on for your Project, Publishers, and Owners will be able to Force Apply their Change Request and bypass the review process. This feature is intended to reduce friction for users who have publishing permissions. 
+
+The button `Force Apply` will appear on the Change Request modal for these users. 
+
+If a user chooses to `Force Apply` then they will required to still include a change description, however they cannot select other reviewers.
```
Comment (suthar26) [inline docs/extras/approval-workflows/approval-workflows.md:60]:
```suggestion
If a user chooses to `Force Apply` then they are required to include a change description, however they cannot select other reviewers. 
```

File: docs/extras/approval-workflows/approval-workflows.md [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+2. Once on the Feature Page, click `Review Change Request` to open up the Review Change Request modal. 
+
+![Approve & Apply Change Request](/july-2024-approve-apply-change-request.png)
+
+3. If you `Approve and Apply Changes`, the proposed changes will be saved immediately. 
+4. If you `Reject` the Change Request, you will need to provide a reason for the rejection that will be sent back to the requester via an email notification. 
+
+![Reject Change Request](/july-2024-reject-change-request.png)
+
+*Note: Anyone with a Publisher, or Owner can approve a Change Request, regardless of whether or not their review has been requested.
```
Comment (suthar26) [inline docs/extras/approval-workflows/approval-workflows.md:76]: this is not showing up properly and maybe should be changed with `:::info`

File: docs/extras/approval-workflows/approval-workflows.md (hunk 2 above)
Comment (katemacfarlane) [reply docs/extras/approval-workflows/approval-workflows.md:39]: reworded!

**Review by jsalaber — APPROVED** [review]
